package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "oci-grafana-logs",
   "private": true,
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
There is also a plugin version number in the `src/plugin.json` file that needs to be updated.

Reply:
modified